web/modules/custom/dpl_fbi/src/Plugin/Field/FieldType/CqlSearchItem.php

```diff
@@ -82,6 +94,30 @@ public static function schema(FieldStorageDefinitionInterface $field_definition)
         'description' => 'CQL search string.',
         'length' => 16000,
       ],
+      'location' => [
+        'type' => 'text',
+        'length' => 1024,
+        'not null' => FALSE,
+        'description' => '"Location" search filter',
+      ],
+      'sublocation' => [
+        'type' => 'text',
+        'length' => 1024,
+        'not null' => FALSE,
+        'description' => '"Sub-location" search filter',
+      ],
+      'sort' => [
+        'type' => 'varchar',
+        'length' => 128,
+        'not null' => FALSE,
+        'description' => 'The chosen search sort',
+      ],
+      'onshelf' => [
+        'type' => 'int',
+        'length' => 1,
+        'not null' => FALSE,
+        'description' => '"On shelf" search filter',
+      ],
```

Comment:
Please consider if we need to do something to make these schema changes apply to already installed sites.

Reply:
:+1: Note that subsequent deployments to your PR now fails: https://ui.lagoon.dplplat01.dpl.reload.dk/projects/dpl-cms/dpl-cms-pr-2507/deployments/lagoon-build-pi873u. I think it could be related.